internal/hammer/diff.go

```diff
@@ -197,6 +197,7 @@ func (g *Generator) GenerateDDL() DDL {
 		ddl.AppendDDL(g.generateDDLForConstraints(fromTable, toTable))
 		ddl.AppendDDL(g.generateDDLForRowDeletionPolicy(fromTable, toTable))
 		ddl.AppendDDL(g.generateDDLForCreateChangeStream(g.from, toTable))
+		ddl.AppendDDL(g.generateDDLForAlterIndex(fromTable, toTable))
```

Comment:
I think this line should be defined after generateDDLForCreateIndex, unless there's a specific reason not to.